src/core/org/apache/jmeter/gui/LoggerPanel.java

```diff
@@ -90,6 +91,21 @@ private JTextArea init() { // WARNING: called from ctor so must not be overridde
             jSyntaxTextArea.setLineWrap(false);
             jSyntaxTextArea.setLanguage("text");
             jSyntaxTextArea.setMargin(new Insets(2, 2, 2, 2)); // space between borders and text
+            int fontSize = jSyntaxTextArea.getFont().getSize();
+            increase = new JMenuItem(String.format(JMeterUtils.getResString("font.increase"), (fontSize + 1)));
```

Comment:
The parenthesis around `fontSize + 1` is not needed and should be removed.
The other scaling used in JMeter uses a multiplication scheme. Perhaps this one should follow the other one.